templates/search/blocks/search_results__list-card.html

```diff
@@ -18,7 +18,7 @@ <h3 class="tna-heading-m">
                         <span class="sr-only">Reference number {{ record.reference_number }}: </span>
                     {% endif %}
                 {% endif %}
-                {{ record.summary_title }}
+                {{ record.summary_title|truncatechars:250 }}
```

Comment:
I wonder if something like this might be better? (Code not tested)

```suggestion
                {% if record.summary_title | length > 250 }}
                    {{ record.summary_title | truncatechars:250 }}&hellip;
                {% else %}
                    {{ record.summary_title }}
                {% endif %}
```

Reply:
The filter adds the ellipsis automatically https://docs.djangoproject.com/en/5.0/ref/templates/builtins/#truncatechars